src/qt/combobox.cpp

```diff
@@ -36,6 +38,30 @@ wxQtComboBox::wxQtComboBox( wxWindow *parent, wxComboBox *handler )
             this, &wxQtComboBox::editTextChanged);
 }
 
+void wxQtComboBox::showPopup()
+{
+    wxComboBox *handler = GetHandler();
+    if( handler )
```

Comment:
Is it really normal/expected to not have a handler here?

Reply:
@vadz ,
Probably not.
I'm just cautious here, but the check can probably be removed.